docs/integrations/aws/index.md

```diff
@@ -32,8 +32,92 @@ In additional to runtime libraries installed by the Serverless Extension, an ext
 Lambda Extension is used to forward logs, metrics, and traces to
 Serverless Console. This component is independent from the language runtime.
 
-### Configuring the Serverless Console Extension
+## Configuring the Serverless Console Extension
 [Serverless Framework](../index.md) is the easiest way to instrument your app or
 service with the Serverless Console Extension. If your not using 
 Serverless Framework you'll need to build, and deploy the extension 
 yourself for each of your Lambdas functions. 
+
+### Pre-requisites
+Before you get started make sure you have the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+and a recent version of [Node.js](https://nodejs.dev/learn/how-to-install-nodejs) and a 
+[Serverless Console Organization](https://console.serverless.com?ref_website=https%3A%2F%2Fwww.serverless.com%2Fconsole%2Fdocs%2F) and [jq]().
+
+### 1.Clone the repo and install Node modules
+Once you have everything you need you'll need to clone this repo
+and install node modules.
+
+```text
+git clone https://github.com/serverless/console.git
+cd console
+npm install
+```
+
+### 2.Build the extension locally
+Next you'll need to run a set of build commands to build
+the internal and external extensions.
+
+```
+cd node/packages/aws-lambda-otel-extension/external/otel-extension-external
+npm install
+cd ../../internal/otel-extension-internal-node
+npm install
+cd ../../
+npm run build
+```
+This will generate an artifact in `dist/extension.zip`
+
+### 3.Publish the extension to your AWS Account
+Once you have built the extension, you can publish it 
+to your AWS account.
+
+```text
+aws lambda publish-layer-version \
+   --layer-name sls-otel-extension \
+   --zip-file fileb://./dist/extension.zip 
```

Comment:
Is this supposed to be `fileb://` and not `file://`?

Reply:
oddly the fileb is actually what you want. i thought it was a typo when @medikoo shared it with me originally.

Reply:
Oh nice lol AWS just wants the be different I guess 😆